apps/andi/assets/js/app.js

```diff
@@ -0,0 +1,18 @@
+// We need to import the CSS so that webpack will load it.
+// The MiniCssExtractPlugin is used to separate it out into
+// its own CSS file.
+import normalize_css from 'normalize.css'
+import scss from "../css/app.scss"
+
+// webpack automatically bundles all modules in your
+// entry points. Those entry points can be configured
+// in "webpack.config.js".
+//
+// Import dependencies
+//
```

Comment:
Feels like we can delete most of these comments.

Reply:
Again, I'm disappointed to see all of these conversations resolved without any response or explanation.